manuscript/book.md

```diff
@@ -929,13 +917,13 @@ function validate(values) {
 
 This function is very long, with lots and lots of repetitive boilerplate code. It’s really hard to read and maintain. Sometimes validations for the same field aren’t grouped together.
 
-But if we look closer, there are just four three unique validations:
+But if we look closer, there are just three unique validations:
 
 - a required field (in some cases leading and trailing whitespace is ignored, in some not — hard to tell whether it’s intentional or not);
 - maximum length (always 80);
 - no spaces allowed.
 
-First, let’s extract all validation logic into their own functions, so we could reuse them later:
+First, let’s extract all validation logic into its own functions so we can reuse them later:
```

Comment:
Maybe 

```suggestion
First, let’s extract all validations into their own functions so we can reuse them later:
```

My head explodes because of this mix of singulars and plurals 😑